profiles/source/dictionary/schema.yaml

```diff
@@ -81,6 +81,24 @@ tableSchema:
           }
     missingValues:
       "$ref": "#/definitions/tableSchemaMissingValues"
+    name:
+      "$ref": "#/definitions/name"
+    title:
+      "$ref": "#/definitions/title"
+    description:
+      "$ref": "#/definitions/description"
+    homepage:
+      "$ref": "#/definitions/homepage"
+    created:
+      "$ref": "#/definitions/created"
+    version:
+      "$ref": "#/definitions/version"
+    keywords:
+      "$ref": "#/definitions/keywords"
+    contributors:
```

Comment:
Four properties from Data Package are not carried over to Schema. I'm fine with waiting for use cases, but wanted to know your opinion on these @pierrecamilleri:

- `id`: potentially useful. In my use case, we opted for `path` instead (the URL where the schema is available), which could be useful to add.
- `licenses`: likely a bit tricky to inherit, would not add for now
- `image`: probably seldom useful
- `sources`: not really applicable here

Reply:
Sorry I forgot to reply on this one, I agree on every point with you !
We use the `name` property as an ID in our [schema catalog](https://schema.data.gouv.fr/) and we have on our use case no need for an additional field.